### ISSUE TITLE
expose Dataset.report as a command line function

### ISSUE THREAD

**Issue by SimonGreenhill** [lexibank/pylexibank#81]
This would allow the report to be updated without running install/makecldf etc. It's also currently tightly coupled to `_install`, so it'd be good to pull this apart as well.

**Comment by LinguList** [comment]
Yes, when we start enhancing this further, e.g, adding more info on cognate sets, etc., this will come in handy. I love the `lexibank makecldf dataset` function. Even better, if one could ask for a short report, specifically while working on the report function.

**Comment by SimonGreenhill** [comment]
perfect, thanks @xrotwang
